### PR TITLE
Made the 404 page more informative.

### PR DESCRIPTION
**Pull request by sakkshm**
Fixes https://github.com/gatsbyjs/store.gatsbyjs.org/issues/290

### CODE REVIEW

File: src/pages/404.js [hunk 1]
```diff
@@ -3,14 +3,21 @@ import { Heading, Text, TextContainer } from '../components/shared/Typography';
 
 const NotFoundPage = () => (
   <TextContainer>
-    <Heading>Whoops - That Page Doesn’t Exist (404)</Heading>
```
Comment (prvnbist) [inline src/pages/404.js]: 'That' here refers to the page that threw the 404, 'This' referes to the 404 page itself.

File: src/pages/404.js [hunk 2]
```diff
@@ -3,14 +3,21 @@ import { Heading, Text, TextContainer } from '../components/shared/Typography';
...
       If you think this is an error or ended up at this page by following a
       link, please{' '}
       <a href="https://github.com/gatsbyjs/gatsby/issues/new">open an issue</a>{' '}
       to let us know.
+      {"\n"}
+      <a href="https://store.gatsbyjs.org">Home - Swag store</a>{"\n"}
+ {"\n"}
+      <a href="https://www.gatsbyjs.org">Home - Gatsby JS</a>{"\n"}
+ {"\n"}
+    Contribute to Gatsby JS on {' '} <a href="https://www.gatsbyjs.org">Github</a>{"\n"}
```
Comment (prvnbist) [inline src/pages/404.js]: You should use `<br>` when introducing new lines but css styling is preferred over inserting new lines manually. 
The page already has a link to go back to homepage in the top right. The purpose of a 404 page is to provide the user meaningful suggestions that falls within the context of the website and not to redirect them away to another website.

File: src/pages/404.js (hunk 1 above)
Comment (sakkshm) [reply src/pages/404.js]: This is a common feature I have seen in many websites 
This is not to deviate users from the website but to give them a small list of links that can help them go to a useful page!

**Comment by sakkshm** [comment]
I see this link list as a common feature on websites 
This is not to deviate users from the website but to give them a small list of links that can help them go to a useful page!

File: src/pages/404.js [hunk 3]
```diff
@@ -3,14 +3,21 @@ import { Heading, Text, TextContainer } from '../components/shared/Typography';
...
-    <Heading>Whoops - That Page Doesn’t Exist (404)</Heading>
+    <Heading>Whoops! - This Page Doesn’t Exist (404)</Heading>
     <Text>
       Looks like the page you requested either doesn’t exist or has been moved.
       If you think this is an error or ended up at this page by following a
       link, please{' '}
       <a href="https://github.com/gatsbyjs/gatsby/issues/new">open an issue</a>{' '}
       to let us know.
+      {"\n"}
+      <a href="https://store.gatsbyjs.org">Home - Swag store</a>{"\n"}
```
Comment (sidharthachatterjee) [inline src/pages/404.js]:
```suggestion
      <a href="https://store.gatsbyjs.org">Swag Store homepage</a>
```

File: src/pages/404.js [hunk 4]
```diff
@@ -3,14 +3,21 @@ import { Heading, Text, TextContainer } from '../components/shared/Typography';
...
   <TextContainer>
-    <Heading>Whoops - That Page Doesn’t Exist (404)</Heading>
+    <Heading>Whoops! - This Page Doesn’t Exist (404)</Heading>
     <Text>
       Looks like the page you requested either doesn’t exist or has been moved.
       If you think this is an error or ended up at this page by following a
       link, please{' '}
       <a href="https://github.com/gatsbyjs/gatsby/issues/new">open an issue</a>{' '}
       to let us know.
+      {"\n"}
```
Comment (sidharthachatterjee) [inline src/pages/404.js]:
```suggestion

```

File: src/pages/404.js [hunk 5]
```diff
@@ -3,14 +3,21 @@ import { Heading, Text, TextContainer } from '../components/shared/Typography';
...
+      <a href="https://store.gatsbyjs.org">Home - Swag store</a>{"\n"}
+ {"\n"}
```
Comment (sidharthachatterjee) [inline src/pages/404.js]:
```suggestion

```

File: src/pages/404.js [hunk 6]
```diff
@@ -3,14 +3,21 @@ import { Heading, Text, TextContainer } from '../components/shared/Typography';
...
+      <a href="https://www.gatsbyjs.org">Home - Gatsby JS</a>{"\n"}
```
Comment (sidharthachatterjee) [inline src/pages/404.js]:
```suggestion
      <a href="https://www.gatsbyjs.org">GatsbyJS homepage</a>
```

File: src/pages/404.js [hunk 7]
```diff
@@ -3,14 +3,21 @@ import { Heading, Text, TextContainer } from '../components/shared/Typography';
...
+ {"\n"}
```
Comment (sidharthachatterjee) [inline src/pages/404.js]:
```suggestion

```

**Comment by sidharthachatterjee** [comment]
@sakkshm Thank you for opening this! While I _definitely_ see the value with adding a couple of links in the 404 page for the user, the new lines are problematic and I'd prefer we style this content in CSS. 

Let's close this for now and discuss this further in the issue you've very helpfully opened and take it from there? 

Thank you one again for starting this conversation though. We really appreciate it 💜

**Pull request closed without merging**